templates/vue-starter-template/app/components/FrontendNavigationPage.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import { useCmsHead } from "@/composables/useCmsHead";
+import type { Ref } from "vue";
+import { useCategorySearch } from "#imports";
+import type { Schemas } from "#shopware";
+
+const props = defineProps<{
+  navigationId: string;
+}>();
+
+const { search } = useCategorySearch();
+const route = useRoute();
+const { buildDynamicBreadcrumbs } = useBreadcrumbs();
+const { apiClient } = useShopwareContext();
+const errorDetails = ref();
+
+const { data, error } = await useAsyncData(
+  `cmsNavigation${props.navigationId}`,
+  async () => {
+    const responses = await Promise.allSettled([
+      search(props.navigationId, {
+        withCmsAssociations: true,
+        query: {
+          ...route.query,
+        },
+      }),
+      apiClient
+        .invoke("readBreadcrumb get /breadcrumb/{id}", {
+          pathParams: {
+            id: props.navigationId,
+          },
+        })
+        .catch(() => {
+          console.error("Error while fetching breadcrumbs");
+        }),
```

Comment:
let's not do this in allSettled, let's catch errors after the resolve

Reply:
done

Reply:
I still see the "catch" here